Rules/TDM/mapcycle.cfg

```diff
@@ -1,61 +1,79 @@
 			Maps/Official/TDM/RattleCage.png;
 			Maps/Official/TDM/LonelyTreasure.png;
 			Maps/Official/TDM/GamblersDen.png;
 			Maps/Official/TDM/CraggyOutlook.png;
-			Maps/Official/TDM/FortMercury_Revised.png;
+			Maps/Official/TDM/PressureCooker.png;
+			Maps/Official/TDM/MonsteriTDM1.png;
+			Maps/Official/TDM/MonsteriTDM2.png;
+			Maps/Official/TDM/Dichotomy.png;
+			Maps/Official/TDM/Bastille.png;
+			Maps/Official/TDM/ArchPoint.png;
+			Maps/Official/TDM/HergestRidge.png;
+				# LorderPlay
+			Maps/Official/TDM/LorderPlay_Pandoras_box.png;
 				# Lavalord
-			Maps/Official/TDM/LavalordTDM1.png;
 			Maps/Official/TDM/LavalordTDM2.png;
-				# Gurin
-			Maps/Official/TDM/GurinTDM3.png;
 				# Biurza
 			Maps/Official/TDM/Biurza_Retroactive_Interference.png;
-			Maps/Official/TDM/Biurza_Black_Dog.png;
-			Maps/Official/TDM/Biurza_Abrupt_Ending.png;
-			Maps/Official/TDM/Biurza_Death_Penalty.png;
+			Maps/Official/TDM/Biurza_Spotlight_Kid.png;
+			Maps/Official/TDM/Biurza_Cock_Fight.png;
 				# Ni
 			Maps/Official/TDM/Ni_Watery.png;
 			Maps/Official/TDM/Ni_TrainingGround_Revised.png;
-			Maps/Official/TDM/Ni_Ashes.png;
+			Maps/Official/TDM/Ni_GetisArena.png;
 				# Ferrezinhre
 			Maps/Official/TDM/Ferrezinhre_Reaver.png;
 			Maps/Official/TDM/Ferrezinhre_Still_Free.png;
-			Maps/Official/TDM/Ferrezinhre_Wish.png;
+			Maps/Official/TDM/Ferrezinhre_CursedWell.png;
 				# 8x
 			Maps/Official/TDM/8x_Virgo.png;
-			Maps/Official/TDM/8x_Dirtkissing.png;
-			Maps/Official/TDM/8x_Cascade1.png;
 			Maps/Official/TDM/8x_Mandala.png;
-			Maps/Official/TDM/8x_TDM1.png;
 			Maps/Official/TDM/8x_TDM2.png;
 			Maps/Official/TDM/8x_Stringer_Revised.png;
 				# FG
 			Maps/Official/TDM/FG_Clash.png;
 			Maps/Official/TDM/FG_Bunker_Revised.png;
+			Maps/Official/TDM/FG_Descent.png;
+			Maps/Official/TDM/FG_Break.png;
+			Maps/Official/TDM/FG_Enclosure.png;
+			Maps/Official/TDM/FG_SpaceInvader.png;
 				# SGG
 			Maps/Official/TDM/super_gaming_geek_Trilevel.png;
 			Maps/Official/TDM/super_gaming_geek_Nook.png;
-			Maps/Official/TDM/super_gaming_geek_ChokePoint.png;
 				# bunnie
 			Maps/Official/TDM/HomekBunnie_StrusBorzy.png;
 			Maps/Official/TDM/SzczurWodny.png;
-				#Yeti5000
-			Maps/Official/TDM/Yeti5000_Upsurge_Revised.png;
-			
+			Maps/Official/TDM/bunnie_Wojciech_Jaruzelski.png;
+				# Yeti5000
+			Maps/Official/TDM/Yeti5000_Upsurge.png;
+				# Vans
+			Maps/Official/TDM/Vans_Spelunca.png;
+			Maps/Official/TDM/Vans_Corruptio.png;
+			Maps/Official/TDM/Vans_Ziraeth.png;
+				# Pineapple
+			Maps/Official/TDM/Pineapple_Vestige.png;
+				# Gurin
+			Maps/Official/TDM/GurinTDM1.png;
+			Maps/Official/TDM/GurinTDM2.png;
+				# JoanOfArc
+			Maps/Official/TDM/Eluded_JoansMansion.png;
+				# CrocoDile
+			Maps/Official/TDM/CrocoDile_FireBird.png;
+				# Cohen
+			Maps/Official/TDM/Cohen_Sanctuary.png;
+				# SJD360
+			Maps/Official/TDM/SJD360_CrowbarsSwitchblades.png; 
+				# mcrifel
+			Maps/Official/TDM/mcrifel_Ruin.png;
```

Comment:
None of the `_v2` variants are being used. The `gamemode.cfg` should be edited.

Reply:
you meant `mapcycle.cfg`?

Reply:
yes, whoops